packages/List/src/list-item.scss

```diff
@@ -48,8 +48,8 @@
   --ids-list-item-checkbox-border: #{tokens.$grey-400};
   --ids-list-item-checkbox-border-hover: #{tokens.$grey-800};
   --ids-list-item-checkbox-border-checked: #{tokens.$electric-blue-500};
-  --ids-list-item-checkbox-check-width: 4px;
-  --ids-list-item-checkbox-check-height: 8px;
+  --ids-list-item-checkbox-check-width: .4rem;
+  --ids-list-item-checkbox-check-height: .4rem;
```

Comment:
Is it normal this value changed in the process?

Reply:
shoot, no good catch! it should be .8rem